vars/buildPlugin2.groovy

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env groovy
+
+/**
+ * Simple wrapper step for building a plugin
+ */
+def call(Map params = [:]) {
+    def platforms = params.containsKey('platforms') ? params.platforms : ['linux', 'windows']
```

Comment:
Is there an advantage to this syntax over `def platforms = params.platforms ?: ['linux', 'windows']`?

(just wondering)